src/main/java/org/mqttbee/api/mqtt/MqttClientBuilder.java

```diff
@@ -53,20 +54,44 @@ public MqttClientBuilder withIdentifier(@NotNull final MqttClientIdentifier iden
     }
 
     @NotNull
-    public MqttClientBuilder forServerHost(@NotNull final String host) {
+    public MqttClientBuilder withServerHost(@NotNull final String host) {
         this.serverHost = host;
         return this;
     }
 
     @NotNull
-    public MqttClientBuilder forServerPort(final int port) {
+    public MqttClientBuilder withServerPort(final int port) {
         this.serverPort = port;
         return this;
     }
 
     @NotNull
-    public MqttClientBuilder usingSSL(final boolean usesSSl) {
-        this.usesSSL = usesSSl;
+    public MqttClientBuilder withWebsocketConfig(MqttWebsocketConfig mqttWebsocketConfig) {
```

Comment:
Remove this method as usingWebSockets(MqttWebsocketConfig) is the right one

Reply:
done